src/Regex.cpp

```diff
@@ -37,7 +50,8 @@ FindRegexMatch::FindRegexMatch(const boost::regex &regexImpl, const std::string
             ++i;
         for (; i != matchResults.end(); ++i) {
             if (i->matched) {
-                RegexSubmatch s = {*i, i->first - expression.begin()};
+                std::ptrdiff_t bytePosition = i->first - expression.begin();
+                RegexSubmatch s = {*i, codepointOffset(expression, bytePosition)};
```

Comment:
Now that this constructor throws, please adjust its declaration, cf. `internal/Table.hpp`: `addColumn(·)`
Also 🐦: `bytePosition`, see above.